### ISSUE TITLE
No playback of media and endless loading of Jellyfin content.

### ISSUE THREAD

**Issue by Elite-Alien** [jeffvli/feishin#682]
### Expected Behavior

Media should play as expected.

### Current Behavior

Feishin does not play any media at all. Both in "Web" and "MPV".  This behavior happens on three different versions of Feishin. I've tried the Appimage, AUR Package, and Docker Compose.

### Steps to Reproduce

1) Open the application & login to my server.
2) Connect with user and click on a album to have a endless loading screen.
3) Press play on a song and the song never starts. It sits at 0 none stop.
![feishin](https://github.com/user-attachments/assets/c7f2fc38-a8a1-40cf-914b-831f81c9d104)

### Context

I can't play any content at all or view content or artist pages period. I can see the album artwork and artist images. However, any sub-menu or content beyond this page goes nowhere.
![feishin1](https://github.com/user-attachments/assets/8a127f70-8ace-4261-84f9-9234336905fe)
![feishin2](https://github.com/user-attachments/assets/81f9c0aa-d81d-41ba-bd34-1e2644948d80)

I have used both the appimage downloaded directly and the one from the AUR with zero success. I have also used the AUR Feishin-bin with the same results, and used Docker-compose with the provided yaml information to have the same result. 

I've used the "Web" setting to have the same issue. I've also switched to MPV and selected the path to my MPV the same result, but also get warnings of it not able to load the file or stream. Though Jellyfin's Dashboard states it's playing and continues to count up on the playtime despite Feishin saying 0 and playing no audio. The direct play states no issues too on the dashboard, so there is no attempt to transcode the media.
![feishin3](https://github.com/user-attachments/assets/fa0351e9-37c5-4a27-b495-acf61b87381a)
![feishin4](https://github.com/user-attachments/assets/07039824-065c-4709-9fd6-4b242ad05c8b)
![feishin5](https://github.com/user-attachments/assets/26ed860d-9a75-4a2b-8a78-bd317591de1a)
![Feishin6](https://github.com/user-attachments/assets/ee7c3bb6-9478-422e-a586-44fe41c25e6c)
![feishin7](https://github.com/user-attachments/assets/3f92d661-3afc-462e-8dc8-fc628f8be4e8)

### Application version

V0.71-1 & V0.7.1-2

### Operating System and version

Arch Linux (kernel 6.10-1) & Ubuntu Server 24.04 LTS (Docker)

### Server and Version

Jellyfin v10.9.8 (server & web)

**Comment by jeffvli** [comment]
You need to open the devtools from the app filemenu and check the console and network tabs for errors on the pages.

It looks like a permissions issue on first glance but seeing the logs and network requests should give you the info you need to debug it.

**Comment by Elite-Alien** [comment]
The account always has had proper permissions. I can access everything from the web interface on Jellyfin just fine. I've listened to my music, watched my content, et cetera on the account fine for years. Both on browser, Jellyfin-Media-Player, and other applications. Jellyfin gives no such errors and the logs of problems. I tried another application that was similar and it loaded fine. 

As for the network tab, well it looks like it's working, but there is still no playback from Feishin, it remains stuck with a non-moving progress bar.
![Feishin](https://github.com/user-attachments/assets/ede5dd48-2905-4fff-910d-90e640366189)

The progress bar just does not move and the sound is not there despite there being a entry in the sound mixer. It lists Feishin as Chromium and appears like this as soon as I hit "play". So the audio is not muted either.
![Feishin-Sound](https://github.com/user-attachments/assets/b8d9904d-bebd-43df-9eb3-061ba00bb2f2)

I appreciate the help and if there is any other suggestions I will try them. I really don't understand why Feishin can not play my music. While everything else can with the same user account(s). I've tried Preserve, Finamp (flatpak), a few others, and even on some applications via my phone and it all works. Yet Feishin has left me boggled. 

What permission does this need the other applications do not? I've even used the admin account for my Jellyfin server on Feishin and it will not play. So I would think I had all the permissions to access my files. Though if there is a plugin or some kind of thing I need for Jellyin I can try it. Like do I need DLNA? I don't see why I would and no means to plug that into Feishin, but perhaps I missed something?

**Comment by Elite-Alien** [comment]
I don't know what fixed it. Though Jellyfin updated to 10.9.9 and I figured I would give it another try. It's now just magically working. I changed nothing, did nothing, and it just works now. What the heck? I really have no idea what was going on, but hopefully no updates break it again.

Thanks to those that took a look at my problem and to Jeffvli for attempting to help. Much appreciated. I wish I knew what to say the problem was, but I am just here clueless and the logs showed no problems. Be it before or after.
